apps/gallery/style/gallery_tablet.css

```diff
@@ -339,7 +339,7 @@ body.fullscreenView #fullscreen-back-button-large {
  */
 #overlay,
 #confirm-dialog {
-  background: rgba(51, 51, 51, 0.6);
+  background: rgb(51, 51, 51);
```

Comment:
I think this is more concisely expressed as #333, if I did the conversion correctly.